lib/devices/ios/ios.js

```diff
@@ -1036,6 +1036,18 @@ IOS.getDeviceStringFromOpts = function (opts) {
       'iPad Simulator (9.0)': 'iPad 2 (9.0)',
       'iPad Simulator (9.1)': 'iPad 2 (9.1)',
       'iPad Simulator (9.2)': 'iPad 2 (9.2)',
+      'iPhone Simulator (8.1)': 'iPhone 6s (8.1)',
```

Comment:
That does not matter. For a single key, like `iPhone Simulator (8.1)`, we only want to map to a single device, like `iPhone 6 (8.1)`. For you to be able to use, say, `iPhone 6s Plus` in 9.1, there needs to exist an entry on the left side, for `iPhone 6s Plus (9.1)`, that maps to a valid device. But each key (i.e., left side of the "x: y" assignment) needs to be **unique**.